### DISCUSSION TITLE
500 Internal Server Error when creating an account

### DISCUSSION THREAD

**Discussion by Hyeonsik** [redivis/meta#249] [Q&A]
When I attempted to create an account on the Stanford University data portal through Redivis, I encountered a '500 Internal Server Error.' This error occurred when I tried to sign up using Google and when selecting an institution. I hope for a quick fix. Thank you.

**Comment by imathews** [comment]
Hi @Hyeonsik – I'm quite sorry for the delay here. Can you please confirm whether this issue is still occurring for you?

**Reply by Hyeonsik** [reply]
Hi @imathews . 

Unfortunately, I'm still encountering the same error when trying to create my account.

Could you please advise on the next steps?

Here is the error.
Oops! An Error Occurred
The server returned a "500 Internal Server Error".
Something is broken. Please let us know what you were doing when this error occurred. We will fix it as soon as possible. Sorry for any inconvenience caused.


Thank you.

Best regrads,
Hyeonsik.

**Reply by imathews** [reply]
Hi @Hyeonsik – I was able to identify an issue in our error parsing here, that caused this fairly unhelpful error message. Could you try it one more time? I think you will still encounter an error, but it should hopefully contain a bit more information, and we can continue to triage from there.

**Reply by Hyeonsik** [reply]
Thanks @imathews . But the same error message still appears.

**Reply by sfmcintyre** [reply]
Hi @Hyeonsik - thanks so much for your engagement as we work to resolve this issue. I'll let you know when we deploy a fix!

**Reply by sfmcintyre** [reply]
Hi @Hyeonsik – please try the following: navigate to your Google account's [3rd party apps & services dashboard](https://security.google.com/settings/security/permissions), and check if you have any items labeled **Redivis** in the list of applications. If so, navigate to the item and **Delete all connections you have with Redivis**. After this, head to Redivis to restart the [Create an account](https://redivis.com/?createAccount) flow, and (hopefully) create your account!

If the above doesn't work, I'd love to hop on a call to troubleshoot further – I'm available all day tomorrow (Friday 8/29) and can be reached at [sean@redivis.com](mailto:sean@redivis.com). Authorization issues with Google accounts are difficult to re-create, so I'd appreciate your time to see exactly what's happening on your end and help you get up and running!
